drv/sidecar-seq-server/src/main.rs

```diff
@@ -292,6 +296,8 @@ fn main() -> ! {
 
     ringbuf_entry!(Trace::FpgaInit);
 
+    server.mainboard_controller.reset(); // force reinit
```

Comment:
Is this related to the aux flash changes? I have minimal context for this task so not sure why this would be needed.

Reply:
Nope, this was for debug and related to #769.  I've removed it for now; next time we update the mainboard FPGA image, we'll need the checksum stuff to be available (otherwise powered-on Sidecars won't know to update their images).

Reply:
Yeah this has been on my TODO for too long and I got bitten by this the other day (updating the FPGA bitstream but the SP not reloading it on boot).